analyzer/src/main/kotlin/managers/utils/PackageManagerDependencyHandler.kt

```diff
@@ -58,23 +55,20 @@ class PackageManagerDependencyHandler(
             PackageReference(
                 id = Identifier(
                     type = TYPE,
-                    namespace = "",
-                    name = jsonMapper.writeValueAsString(
-                        PackageManagerDependency(
-                            packageManager,
-                            definitionFile,
-                            scope,
-                            linkage
-                        )
-                    ).encode(),
-                    version = ""
+                    namespace = packageManager,
```

Comment:
Commit message: Can you explain the motivation behind this change, in particular why is the new approach better/simpler than the old one?